pkg/operator/controller/gateway-service-dns/controller.go

```diff
@@ -42,7 +42,7 @@ const (
 	// deployments that it creates for gateways that it manages.  Istio uses
 	// this label in the selector of any service that it creates for a
 	// gateway.
-	gatewayNameLabelKey = "istio.io/gateway-name"
+	gatewayNameLabelKey = "gateway.networking.k8s.io/gateway-name"
```

Comment:
Is this an OSSM3 change?

Reply:
Yes.  I don't know whether it's OSSM itself or Istio, but the change came with the bump from OSSM 2 to OSSM 3.